app/services/org/create_created_plan_service.rb

```diff
@@ -6,24 +6,54 @@ def call(org = nil)
 
         orgs.each do |org|
           OrgDateRangeable.split_months_from_creation(org) do |start_date, end_date|
-            create_count_for_date(start_date: start_date, end_date: end_date, org: org)  
+            create_or_update(start_date: start_date, end_date: end_date, org: org)
           end
         end
       end
 
       private
 
+      def users(org)
+        User.where('users.org_id = ?', org.id)
```

Comment:
prefer to see the use of the symbol here: `User.where(org: org)`

Reply:
me too, but with the way you propose ActiveRecord is not smart enough to determine to what table that attribute refers when used through 'merge' spawn method. You have to namespace it, i.e. ```{ users: { org_id: org_id }}```